en/setup/getting_started.md

```diff
@@ -4,11 +4,77 @@ Awesome! You've downloaded a marketing automation tool. That's a great first ste
 
 ## Step 1: Install Mautic
 
-If you have already downloaded the zip from the download page or have installed Mautic through some other source (Softaculous, Bitnami, Digital Ocean etc...) then you have already completed the first step. If not then you will need to upload the Mautic package (a zip file) to your server; unzip the files; and then navigate to that location in your browser. You will find Mautic has a very easy to follow on-screen installation process.
+If you have already downloaded the zip from the download page or have installed Mautic through some other source (Softaculous, Bitnami, Digital Ocean etc...) then you have already completed the first step. If not then you will need to upload the Mautic package (a zip file) to your server; unzip the files; and then navigate to that location in your browser. 
+
+As an example, if your domain is mydomain.com and you have unzipped the file into a folder called 'm', you would then browse to mydomain.com/m to start the installation process.
+
+###Permission errors
+If you see errors when you try to access the installation as below:
+
+![](/setup/media/permissions-error.png)
+
+the problem is likely to be due to the permissions on your files and folders.  
+
+If you have access to the command line, you can run the following commands to reset the file and folder permissions.  Make sure that you are in the directory where Mautic has been installed (using the example above, that would mean executing these commands within /m in your web root folder - you can get there using the `cd` command)
+
+```find . -type d -exec chmod 755 {} \;
+find . -type f -exec chmod 644 {} \;
+chmod -R g+w app/cache/
+chmod -R g+w app/logs/
+chmod -R g+w app/config/
+chmod -R g+w media/files/
+chmod -R g+w media/images/
+chmod -R g+w translations/
+chown -R $USER:www-data .```
```

Comment:
For this kind of code block, I would suggest avoiding use of triple backticks and instead simply indenting the whole thing by 4 columns:

    This is some normal text.

        find . -type f -exec chmod 644 {} \;
        chmod -R g+w app/cache/

    and normal text again.

This will achieve exactly the same rendering effect, but has the advantage of making the non-rendered plain-text file much more readable.  This adheres to Markdown's original design goal, as stated on [the homepage](https://daringfireball.net/projects/markdown/):

> The overriding design goal for Markdown’s formatting syntax is to make it as readable as possible.

However, if you do use triple backticks, bear in mind the following:

- [Any text immediately following the opening backticks on the first line is used by GitHub to indicate the coding language of the rest of the block, so that it can syntax highlight correctly](https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown)
- For single-line code blocks, single backticks are sufficient.  Triple backticks are only needed when the content contains backticks of its own.